### PR TITLE
Create monitoring user on provision

### PR DESCRIPTION
**Pull request by felixhoo**
Create monitoring user in instance handle for UI access

### CODE REVIEW

**Comment by felixhoo** [comment]
This change will cause rabbit-ng node deployment failure. Need to recreate rabbit-ng node

File: ng/rabbit/lib/rabbit_service/rabbit_node.rb [hunk 1]
```diff
@@ -277,17 +298,7 @@ def gen_credentials(instance, user = nil, pass = nil)
       "admin_port"  => instance.admin_port,
       "vhost" => instance.vhost,
     }
-    if user && pass # Binding request
```
Comment (kushmerick) [inline ng/rabbit/lib/rabbit_service/rabbit_node.rb:280]: this is much nicer, good idea.

**Comment by kushmerick** [comment]
+1 -- I think this is ready to merge.

@dsboulder @shalako -- can you please merge 
Also note Felix's comment that this is a breaking change, you need to wipe the rabbitmq-nodes.  i think this is acceptable since NG services are still not really used by anyone.

**Comment by shalako** [comment]
@kushmerick added story for SF team to review and merge

**Pull request merged**